tls_codec/src/quic_vec.rs

```diff
@@ -196,15 +197,45 @@ impl<T: Size> Size for &[T] {
 
 // === Vec<u8> and &[u8]
 
+/// Helper type used for `Debug`-printing.
+///
+/// Wrap a `&[u8]` with this type to append a comment with its hex representation
+/// during `Debug`-printing, i.e., `/* 0x... */`.
+struct SliceU8HexDebug<'a>(pub &'a [u8]);
+
+impl<'a> fmt::Debug for SliceU8HexDebug<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.0.fmt(f)?;
+
```

Comment:
Is this really useful? This will get pretty messy for larger slices. I'd prefer skipping this.

Reply:
Could you elaborate? I'm not sure I understand what you propose to skip.

Reply:
You print things twice right now. And I don't see how that's useful.
The output is, as you also say in the description
```
VLBytes { vec: [0] /* 0x00 */ }
```
while
```
VLBytes { vec: 0x00 }
```
or even better
```
VLBytes: 0x00
```
would be sufficient and less cluttered.

Reply:
Seeing this... let's go with the last option :-) We can revisit later when we think that having both representations -- decimal and hexadecimal -- would be useful.